tests/integration_tests/functional/test_error_code.py

```diff
@@ -29,7 +36,7 @@ def test_enosys_error_code(test_microvm_with_ssh, network_config):
         except OSError:
             break
 
-    if platform.machine() != "x86_64":
-        log_data = test_microvm.log_data
-        assert "Received ENOSYS error because KVM failed to emulate an instruction" in log_data
-        assert "Vmm is stopping." in log_data
+    log_data = test_microvm.log_data
+    assert "Received ENOSYS error because KVM failed to emulate " \
+           "an instruction" in log_data
+    assert "Vmm is stopping." in log_data
```

Comment:
Doesn't this check over here mean that the VMM is killed after logging the expected error? If so, the description at line 16 should specify that the purpose of this test is to have the error logged before exiting the VMM.